INSTALL.txt

```diff
@@ -44,40 +43,36 @@ X-12-ARIMA or X-13ARIMA-SEATS
     appropriate executable on your PATH or set the X12PATH or X13PATH 
     environmental variable to take advantage.
 
-matplotlib >= 1.1
+matplotlib >= 1.3
 
-    http://matplotlib.sf.net/
+    http://matplotlib.org/
 
     Matplotlib is needed for plotting functionality and running many of the 
     examples.
 
-sphinx >= 1.0.0
+sphinx >= 1.3
 
     http://sphinx.pocoo.org/
 
     Sphinx is used to build the documentation.
 
-nose >= 1.0.0
+nose >= 1.3
 
     http://readthedocs.org/docs/nose/en/latest/
 
     Nose is needed to run the tests.
 
-IPython >= 1.0
+IPython >= 3.0
 
     Needed to build the docs.
 
 
-Easy Install
-------------
-
-To get the latest release using easy_install you need setuptools (easy_install):
-
-    http://peak.telecommunity.com/DevCenter/EasyInstall
+Installing
+----------
 
-Then you can do (with proper permissions):
+To get the latest release using git
```

Comment:
latest release using *pip* (not git). You might want to recommend the new `--upgrade-strategy="only-if-needed"` option for recent versions of pip.